frontend/src/metabase/browse/components/BrowseMetrics.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from "ttag";
+
+import NoResults from "assets/img/metrics_bot.svg";
+import { useFetchMetrics } from "metabase/common/hooks/use-fetch-metrics";
+import EmptyState from "metabase/components/EmptyState";
+import { DelayedLoadingAndErrorWrapper } from "metabase/components/LoadingAndErrorWrapper/DelayedLoadingAndErrorWrapper";
+import { Box, Flex, Group, Icon, Stack, Text, Title } from "metabase/ui";
+
+import type { MetricResult } from "../types";
+
+import {
+  BrowseContainer,
+  BrowseHeader,
+  BrowseMain,
+  BrowseSection,
+} from "./BrowseContainer.styled";
+import { MetricsTable } from "./MetricsTable";
+
+export function BrowseMetrics() {
+  const metricsResult = useFetchMetrics();
+  const metrics = metricsResult.data?.data as MetricResult[] | undefined;
+
+  const isEmpty = !metricsResult.isLoading && !metrics?.length;
+
+  return (
+    <BrowseContainer>
+      <BrowseHeader role="heading" data-testid="browse-metrics-header">
+        <BrowseSection>
+          <Flex
+            w="100%"
+            h="2.25rem"
+            direction="row"
+            justify="space-between"
+            align="center"
+          >
+            <Title order={1} color="text-dark">
+              <Group spacing="sm">
+                <Icon size={24} color={"var(--mb-color-brand)"} name="metric" />
```

Comment:
@WiNloSt is this usage of CSS variable correct?

Reply:
`--mb-color-brand` is not the newest semantic color we have. It was there before we have more meaningful semantic colors.

Since we're following [Carbon design](https://carbondesignsystem.com/elements/color/tokens/). I'd suggest in this case we create a new color in https://github.com/metabase/metabase/blob/master/frontend/src/metabase/css/core/colors.module.css.

```css
--mb-color-icon-primary: var(--mb-base-color-brand-40); 
```

this is using [a base color](https://www.figma.com/design/OlyKP09vro7EkQ2ZzXDDla/color-playground?node-id=0-1&t=2TB4tnSmQEU6KZ4H-0) to define a new semantic color.

then you could use it like so

```suggestion
                <Icon size={24} color={"var(--mb-color-icon-primary)"} name="metric" />
```